Resources/Private/Form/SingleLineText.html

```diff
@@ -1,5 +1,12 @@
 {namespace form=Neos\Form\ViewHelpers}
 <f:layout name="Neos.Form:Field" />
 <f:section name="field">
-	<f:form.textfield property="{element.identifier}" id="{element.uniqueIdentifier}" class="{element.properties.elementClassAttribute}" placeholder="{element -> form:translateElementProperty(property: 'placeholder')}" errorClass="{element.properties.elementErrorClassAttribute}" maxlength="{element.properties.maxlength}" />
+	<f:if condition="{element.properties.inputMode} == 'search'">
+		<f:then>
+			<f:form.textfield type="search" property="{element.identifier}" id="{element.uniqueIdentifier}" class="{element.properties.elementClassAttribute}" additionalAttributes="{autocapitalize: element.properties.autoCapitalize}" placeholder="{element -> form:translateElementProperty(property: 'placeholder')}" errorClass="{element.properties.elementErrorClassAttribute}" maxlength="{element.properties.maxlength}" />
+		</f:then>
+		<f:else>
+			<f:form.textfield property="{element.identifier}" id="{element.uniqueIdentifier}" class="{element.properties.elementClassAttribute}" additionalAttributes="{autocapitalize: element.properties.autoCapitalize, inputmode: element.properties.inputMode}" placeholder="{element -> form:translateElementProperty(property: 'placeholder')}" errorClass="{element.properties.elementErrorClassAttribute}" maxlength="{element.properties.maxlength}" />
+		</f:else>
+	</f:if>
```

Comment:
What about
```
<f:form.textfield type="{element.properties.type}" property="{element.identifier}" id="{element.uniqueIdentifier}" class="{element.properties.elementClassAttribute}" placeholder="{element -> form:translateElementProperty(property: 'placeholder')}" errorClass="{element.properties.elementErrorClassAttribute}" maxlength="{element.properties.maxlength}" />
```
instead?

Reply:
Potentially, were it not for the fact that I added 2 attributes (autoCapitalize and inputMode). It looks like the "type" will only be able to support one of those attributes.

Reply:
Ah right, I copied the wrong snippet. But
```html
<f:form.textfield property="{element.identifier}" id="{element.uniqueIdentifier}" class="{element.properties.elementClassAttribute}" type="{element.properties.type}" additionalAttributes="{autocapitalize: element.properties.autoCapitalize, inputmode: element.properties.inputMode}" placeholder="{element -> form:translateElementProperty(property: 'placeholder')}" errorClass="{element.properties.elementErrorClassAttribute}" maxlength="{element.properties.maxlength}" />
```

should work, no?
This way we could also use this for other [input types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types)

Reply:
btw: in most cases a specific `type` attribute renders `inputmode` obsolete AFAIK – but IMO there is no harm in providing both properties